dev/galoy/galoy-regtest-values.yml

```diff
@@ -22,6 +22,11 @@ galoy:
       lnd2loopRpcEndpoint: lnd1-loop.galoy-dev-bitcoin.svc.cluster.local:11010
     cronConfig:
       swapEnabled: true
+    kratosConfig:
```

Comment:
This is another setting which is needed, but not present the default galoy chart values.yaml.
In a follow-up PR we should add the regtest values from here to the default values.yaml so everything can be found in one place and then modified in the dev setup to run on signet for example.